sdk/python/src/dagger/engine/_version.py

```diff
@@ -1,4 +1 @@
-# Code generated by dagger. DO NOT EDIT.
-# flake8: noqa
-
-CLI_VERSION = "0.3.7"
+# Code generated by dagger. DO NOT EDIT.\n\nCLI_VERSION = "0.3.8"\n
```

Comment:
Huh, something must have changed since this last ran.

Fixing by hand for the moment, will make an issue to fix the generation